cpp_utils/project_settings.cmake

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###############################################################################
+# Set settings for project cpp_utils
+###############################################################################
+
+set(MODULE_NAME
+    cpp_utils)
+
+set(MODULE_SUMMARY
+    "C++ library for generic useful methods and classes for Developers.")
+
+set(MODULE_FIND_PACKAGES
+        fastcdr
+        fastrtps
```

Comment:
Why `cmake_utils` is not set here as a dependency?

Reply:
Already discussed: cmake_utils must not be linked, and it is explicitly searched for in the main CMakeLists